packages/react-instantsearch/src/components/Autocomplete.tsx

```diff
@@ -17,14 +38,38 @@ export type AutocompleteProps = {
 };
 
 export function EXPERIMENTAL_Autocomplete({ indices }: AutocompleteProps) {
+  const [isOpen, setIsOpen] = useState(false);
+
   return (
     <Index EXPERIMENTAL_isolated>
-      <SearchBox />
-      {indices.map((index) => (
-        <Index key={index.indexName} indexName={index.indexName}>
-          <Hits hitComponent={({ hit }) => <index.itemComponent {...hit} />} />
-        </Index>
-      ))}
+      <Autocomplete isOpen={isOpen}>
+        <SearchBox onFocus={() => setIsOpen(true)} />
+        <AutocompletePanel isOpen={isOpen}>
+          {indices.map((index) => (
+            <Index key={index.indexName} indexName={index.indexName}>
+              <AutocompleteIndexComponent itemComponent={index.itemComponent} />
```

Comment:
ok for now, but later once we do keyboard, we likely will want it to be a flat list of indices and use connectAutocomplete? not sure tbh but it likely doesn't change much